src/components/cards/ListViewTaskCard.tsx

```diff
@@ -45,6 +45,9 @@ export const ListViewTaskCard = ({
           userSelect: 'none',
           cursor: 'default',
           borderBottom: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
+          ':hover': {
+            bgcolor: (theme) => theme.color.gray[150],
```

Comment:
The bgColor for hover state is not `gray[150]`. Can you please check in Figma once again? 

The actual color is `gray[100]` i.e. `#F8F9FB` with 50% opacity. The equivalent hex color is shown in the video below using Figma color picker. Please follow the same. 
![Screenshot from 2024-07-11 12-23-30](https://github.com/copilot-platforms/tasks-app/assets/38468429/47e21996-6487-438c-aae7-88f7561938ec)

Actual hex value for bgColor using Figma picker - 
[color-picker.webm](https://github.com/copilot-platforms/tasks-app/assets/38468429/bfa7b23b-47fb-4312-8c16-9d6d80095cd1)

Reply:
`#FBFCFD` is the actual hex value.